src/app/cluster-building-blocks/QuieterReporting.h

```diff
@@ -112,6 +112,8 @@ class QuieterReportingAttribute
 {
 public:
     explicit QuieterReportingAttribute(const Nullable<T> & initialValue) : mValue(initialValue), mLastDirtyValue(initialValue) {}
+    // constructor that works with arrays of QuieterReportingAttribute
+    explicit QuieterReportingAttribute() : mValue(Nullable<T>(0)), mLastDirtyValue(Nullable<T>(0)) {}
```

Comment:
why would we start with 0 and not null?

Reply:
Good question, the array of quiest attributes I added for the color control are all not nullable so I wanted to init them at 0. But this constructor could init as null. It would matter for the actual color attributes. I can change it

Reply:
@andy31415 done, if you get a chance please re-review.